lib/reporting/identity_verification_report.rb

```diff
@@ -171,27 +188,31 @@ def data
         # IDEA: maybe there's a block form if this we can do that yields results as it loads them
         # to go slightly faster
         fetch_results.each do |row|
-          event_users[row['name']] << row['user_id']
-
-          if row['name'] == Events::IDV_FINAL_RESOLUTION
-            if row['identity_verified'] == '1'
-              event_users[Results::IDV_FINAL_RESOLUTION_VERIFIED] << row['user_id']
-            end
-            if row['gpo_verification_pending'] == '1'
-              event_users[Results::IDV_FINAL_RESOLUTION_GPO] << row['user_id']
-            end
-            if row['in_person_verification_pending'] == '1'
-              event_users[Results::IDV_FINAL_RESOLUTION_IN_PERSON] << row['user_id']
-            end
-            if row['fraud_review_pending'] == '1'
-              event_users[Results::IDV_FINAL_RESOLUTION_FRAUD_REVIEW] << row['user_id']
-            end
+          event = row['name']
+          user_id = row['user_id']
+          success = row['success']
+
+          event_users[event] << user_id
+
+          case event
+          when Events::IDV_FINAL_RESOLUTION
+            event_users[Results::IDV_FINAL_RESOLUTION_VERIFIED] << user_id if row['identity_verified'] == '1'
+            event_users[Results::IDV_FINAL_RESOLUTION_GPO] << user_id if row['gpo_verification_pending'] == '1'
+            event_users[Results::IDV_FINAL_RESOLUTION_IN_PERSON] << user_id if row['in_person_verification_pending'] == '1'
+            event_users[Results::IDV_FINAL_RESOLUTION_FRAUD_REVIEW] << user_id if row['fraud_review_pending'] == '1'
+          when Events::IDV_DOC_AUTH_IMAGE_UPLOAD
+            event_users[Results::IDV_REJECT_DOC_AUTH] << user_id if success == '0'
```

Comment:
[the doc defines this ](https://docs.google.com/document/d/1O6sw0_83EeOGX0QtUiYoHfm-m0VQCJqYQXgsgXuOU1M/edit#bookmark=id.budguol4yf) as 

> 3.a. Doc auth rejected for non-fraud reasons (i.e. any error code other than "fail" or "attention")

So this currently does not take that into account, I think tomorrow I'll try to dig in and see if we can surface that in the CW query to make it simple